src/main/java/org/openjdk/jextract/impl/Constants.java

```diff
@@ -213,26 +238,16 @@ static Constant ofLiteral(Class<?> type, Object value) {
 
     public List<JavaFileObject> toFiles() {
         currentBuilder.classEnd();
-        List<JavaFileObject> files = new ArrayList<>();
-        files.addAll(constantBuilders.stream()
-                .flatMap(b -> b.toFiles().stream()).toList());
-        return files;
+        return new ArrayList<>(constantBuilders.stream()
+                .map(SourceFileBuilder::toFile).toList());
     }
 
     class Builder extends ClassSourceBuilder {
 
-        Builder(JavaSourceBuilder encl, int id) {
-            super(encl, Kind.CLASS, "constants$" + id);
-        }
+        private static final String MEMBER_MODS = "static final";
 
-        String memberMods() {
-            return kind == ClassSourceBuilder.Kind.CLASS ?
-                    "static final " : "";
-        }
```

Comment:
Note that `Builder`s always have the `CLASS` kind, so we can just use a `static final String` for the modifiers.